sda-commons-server-opentelemetry/docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,148 @@
+# Migration Guide
+
+## Java Version
+ You must use at least Java 8 to use OpenTelemetry. [More info](https://opentelemetry.io/docs/migration/opentracing/#language-version-support)
```

Comment:
Imho that doesn't matter in the open telemetry context because sda-commons already requires Java 8 and v3 will require Java 11.